scout/server/blueprints/cases/controllers.py

```diff
@@ -426,6 +433,69 @@ def vcf2cytosure(store, institute_id, case_name, individual_id):
 
     return (individual_obj['display_name'], individual_obj['vcf2cytosure'])
 
+def gene_variants(store, variants_query, page=1, per_page=50):
```

Comment:
You could write a test for this one as well?

Reply:
Hm, might as well do that using a web page directly, but struggling to figure out how the "client" fixture is set up, to exend it with user etc. Some kind of default it seems?

Reply:
If you are stuck I can try to help you with this, but tomorrow :)

Reply:
🐻 Thank you! Contexts in flask is surprisingly subtle. I have a version with trivial operation tests, based on distilling a large number of web pages and threads, that is working on my side. Uploading now to see what gives in travis.

Reply:
😸 Fun - it really worked. Then we can start adding at least basic operation tests for many more views, also those that require user and institute defined! But not here.